packages/playwright-core/src/server/fetch.ts

```diff
@@ -155,7 +155,7 @@ export abstract class APIRequestContext extends SdkObject {
         return { error: `${fetchResponse.status} ${fetchResponse.statusText}` };
       return { fetchResponse: { ...fetchResponse, fetchUid } };
     } catch (e) {
-      return { error: String(e) };
+      return { error: e instanceof Error ? e.message : String(e) };
```

Comment:
this fixes an issue that the actual error got wrapped as `Error: actual error` and landed by that not in the ideal form in the language ports.

Reply:
I believe we are losing backand's stack trace in this case and it often provides useful information about what's going on (at the very least for us).

Reply:
Seems like it never added the stack trace with the previous implementation: 
![image](https://user-images.githubusercontent.com/17984549/141008392-f0102481-9919-4dbe-bab5-f446ba30eba6.png)